packages/hashi/src/mainClient.js

```diff
@@ -1,8 +1,11 @@
+import { ContentNodeResource } from 'kolibri.resources';
```

Comment:
Flagging that we will want to handle this via dependency injection in the long run - as we will have to use Hashi for previewing HTML5 apps accurately in Studio, so this will break.